Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -185,6 +264,55 @@ class StatusBar extends React.Component<{
    */
   static currentHeight = StatusBarManager.HEIGHT;
 
+  /**
+   * Get the current frame (top and height) of the Status Bar.
+   *
+   * Returns a Promise that resolves to a frame object with a `top` and `height`.
+   */
+  static getCurrentFrame(): Promise<{top: number, height: number}> {
+    return StatusBarManager.getCurrentFrame();
+  }
+
+  /**
+   * Adds an event handler. Supported events:
+   *
+   * - `statusBarFrameDidChange`: Fires when the status bar frame has changed.
+   *   The argument to the event handler is a frame object with the keys:
+   *   - `height`: The current height of the status bar.
+   *   - `top`: How far the top of the status bar is offset from the top of the
+   *     screen. This generally only happens when the iOS In-Call Status Bar is
+   *     visible and pushes the app down instead of changing the size.
+   */
+  static addEventListener(
+    eventName: EventName,
+    handler: Function
+  ): {remove: () => void} {
```

Comment:
AFAIK we can use type: `EmitterSubscription` and return `StatusBarManagerEventEmitter.addListener` directly? That way, we don't have to hold `_subscriptions` and `removeEventListener` at all. I feel like `add/removeListener` is the old pattern in RN and the newest APIs are leaning towards the subscription oriented approach.